cli/.gitignore

```diff
@@ -0,0 +1,8 @@
+dist
+build
+.tox
+env
+dcos_spark.egg-info
+tests/integration/__pycache__
+dcos_spark/data/spark**
+**.pyc
```

Comment:
These are all specific to the CLI though? (this file is originally copied from the CLI repo)